action.yml

```diff
@@ -10,19 +10,30 @@ inputs:
     description: The name of the database to bubild.
     required: true
   install-mock-db-objects:
-    description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-nuget-feed-url must also be set, otherwise an error will occur.
-    default: false
+    description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-nuget-feed-url must also be set, otherwise an error will occur. The accepted values are true and false.
```

Comment:
This is only a suggestion, but can we word it like this:
_The expected value is true or false._

That way it's clear a single value is expected.

Reply:
I'm cool with that!